app/(app)/flames/components/SealSummaryModal.tsx

```
@@ -101,61 +208,223 @@ function StatRow({
+  targetSeconds,
 }: SealSummaryModalProps) {
   const t = useTranslations('flames.seal');
+  const shouldReduceMotion = useReducedMotion();
+  const minutes = Math.floor(elapsedSeconds / 60);
   const rewards = calculateRewards(minutes, level);
 
-  const fuelCount = useCountUp(minutes, open);
-  const sparksCount = useCountUp(rewards.sparks, open);
-  const xpCount = useCountUp(rewards.xp, open);
+  // Pick a contextual subtitle based on fuel percentage
+  const subtitle = useMemo(() => {
```

Biome (2.3.14)

[error] This hook does not specify its dependency on elapsedSeconds.

This dependency is being used here, but is not specified in the hook dependency list.

React relies on hook dependencies to determine when to re-compute Effects.
Failing to specify dependencies can result in Effects not updating correctly when state changes.
These "stale closures" are a common source of surprising bugs.
Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)